README.md

```diff
@@ -1,162 +1,114 @@
+
+_If you wish to contribute to the project, please familiarize yourself with the [Contributing Guide](contributing.md) and the [Code of Conduct](codeOfConduct.md) first._ A comprehensive documentation of how Animal-AI works is also available [here](docs/Technical-Overview.md.md), where you can understand the inner workings of how the environment is built and how it functions (_csharp_ and _Python_ codebases).
+
+The Animal-AI environment and packages are currently tested on **Windows 11**, **Linux**, and **MacOS**, with **Python 3.9.x** support, but **Python 3.6.x+** has been reported to be working also. **Linux distros** are also working and stable. 
 
-## This Repo
 
-This repo contains the AnimalAI environment, some introductory python scripts for interacting with it, as well as the [900 tasks](configs/competition) which were used in the original Animal-AI Olympics competition (and some others for demonstration purposes). Details of the tasks can be found on the [AAI website](http://animalai.org) where they can also be played and competition entries watched.
+## Features
 
-The environment is built using [Unity ml-agents](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) **Release 20-stable (Python version 0.30.0)**. **Unity** (game engine) is the engine used to develop the platform.
+**Interdisciplinary Research Platform:**
+- Facilitates research in human, animal, and artificial cognition.
+- Supports cross-disciplinary studies.
+- Enables comparative assessments among humans, animals, and artificial intelligences.
 
-The AnimalAI environment and packages are currently tested on **Windows 11**, **Linux**, and **Mac**, with minimum **Python 3.8.x**, but Python 3.6+ has reported to be working also. **Linux distros** are also working and stable.
+**Comprehensive AI Environment:**
+- Includes a versatile environment for AI experiments, from basic to advanced configurations.
+- Wrap Unity learning environments as a gym(nasium) environment
+- Wrap Unity learning environments as a PettingZoo environment
+- Support for several Deep Reinforcement Learning algorithms (PPO, SAC, MA-POCA, self-play).
 
-**The Unity Project** for the environment is available [here](https://github.com/Kinds-of-Intelligence-CFI/animal-ai-unity-project).
+**Extensive Task Library:**
+- Multiple example tasks.
+- 900 tasks from the Animal-AI Olympics.
+- Procedural Generation functionality
+- Additional tasks for demonstration and experimentation.
 
+**Unity Game Engine:**
+- Utilizes Unity ml-agents.
+- Leverages Unity game engine for advanced simulation capabilities.
+- Fast and robust wrapper.
 
-## Quick Install (please see Release for latest version of AAI-3)
+**Cross-Platform Compatibility:**
```

Comment:
This seems to contradict line 32 where we say we only support Python 3.9. I would pull this out of features and include an explicit section on what we do support

Reply:
We officially support 3.6 to 3.9 and based on my testing, python 3.6 to 3.9 in between do work. I think we have a choice to make python more flexible by testing on 3.6 to 3.9 instead of just supporting 3.9, which seems limiting.